build/windows/installer/Package.wxs

```diff
@@ -0,0 +1,51 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+    <!-- ToDo use your own keys for Name and Manufacturer, as well as a new UUID for UpgradeCode -->
+    <Package
+        Name="BevyGame"
+        Manufacturer="My Name"
+        UpgradeCode="132bb59d-de4f-4893-a535-93dfd0597731"
+        Version="0.1.0"
+
+        Scope="perUserOrMachine">
+        <MajorUpgrade DowngradeErrorMessage="!(loc.DowngradeError)" />
+
+        <Media Id="1" Cabinet="myapplication.cab" EmbedCab="yes" />
```

Comment:
Should this also have a `ToDo` comment, or where is the `myapplication.cab` used?

Reply:
This is used internally, but I have no idea what it actually does. The name is arbitrary, no TODO needed

Reply:
@janhohenheim If I understand this feature, the goal is to embed the cab file in the MSI installer. I don't know where the myapplication.cab name comes from though.

`<MediaTemplate EmbedCab="yes" />` should do the same thing without us having to explicitly specify the cab's name.

Reply:
Where are we actually creating the cabinet? Is that build automatically from the included directories?